pages/_app.tsx

```diff
@@ -93,12 +93,7 @@ export default function MyApp({ Component, pageProps }: AppProps) {
   );
 
   return (
-    <ThemeProvider
-      attribute="class"
-      defaultTheme="light"
-      enableSystem
-      disableTransitionOnChange
-    >
+    <ThemeProvider attribute="class" defaultTheme="light" enableSystem disableTransitionOnChange>
```

Comment:
sadly my line length for prettier is wider – feel free to reformat or maybe there's .vscode workspace settings that should be standardized

Reply:
no worries, we have an upcoming PR to standardize this :)